src/nccl_ofi_rdma.cpp

```diff
@@ -1696,8 +1695,17 @@ int nccl_net_ofi_rdma_ep_t::ofi_process_cq()
 	nccl_net_ofi_rdma_domain_t *domain_ptr = rdma_endpoint_get_domain();
 	nccl_net_ofi_rdma_device_t *device = domain_ptr->rdma_domain_get_device();
 
-	for (uint16_t rail_id = 0; rail_id != domain_ptr->num_rails; ++rail_id) {
-		nccl_net_ofi_rdma_domain_rail_t *rail = domain_ptr->rdma_domain_get_rail(rail_id);
+	for (uint16_t rail_id = 0; rail_id != this->num_rails; ++rail_id) {
+		nccl_net_ofi_ep_rail_t *rail = this->rdma_endpoint_get_rail(rail_id);
+
+		ret = ofi_process_cq_rail(device, rail);
```

Comment:
I thought we were still going to share the CQ between control and data rails.  Don't love having to poll each CQ twice, because cq polling performance has been a constant thorn in our side.

Reply:
thanks, Brian. I will revisit the cq process function for concurrent accesses and the ep/cq objects.

Reply:
Hi @bwbarrett , I have incorporated your feedback and updated the PR.